ports/plunderislands/Plunder Islands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIR=/$directory/ports/plunderislands/
+CONFDIR="$GAMEDIR/conf/"
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+# Ensure the conf directory exists
+mkdir -p "$GAMEDIR/conf"
+
+# Set the XDG environment variables for config & savefiles
+export XDG_DATA_HOME="$CONFDIR"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+cd $GAMEDIR
+
+# patch game
+[ -f "./gamedata/Plunder Islands.pck" ] && $controlfolder/xdelta3 -d -s "gamedata/Plunder Islands.pck" "patch/plunderislandspatch.xdelta" "gamedata/plunderislandspatched.pck" && rm "gamedata/Plunder Islands.pck"
+
+runtime="frt_3.5.2"
+if [ ! -f "$controlfolder/libs/${runtime}.squashfs" ]; then
+  # Check for runtime if not downloaded via PM
+  if [ ! -f "$controlfolder/harbourmaster" ]; then
+    pm_message "This port requires the latest PortMaster to run, please go to https://portmaster.games/ for more info."
+    sleep 5
+    exit 1
+  fi
+
+  $ESUDO $controlfolder/harbourmaster --quiet --no-check runtime_check "${runtime}.squashfs"
+fi
+
+# Setup Godot
+godot_dir="$HOME/godot"
+godot_file="$controlfolder/libs/${runtime}.squashfs"
+$ESUDO mkdir -p "$godot_dir"
+$ESUDO umount "$godot_file" || true
+$ESUDO mount "$godot_file" "$godot_dir"
+PATH="$godot_dir:$PATH"
+
+export FRT_NO_EXIT_SHORTCUTS=FRT_NO_EXIT_SHORTCUTS
+
+$GPTOKEYB "$runtime" -c "plunderislands.gptk" &
+pm_platform_helper "$runtime"
```

Comment:
`pm_platform_helper` needs to be called with an absolute path.

```suggestion
pm_platform_helper "$godot_dir/$runtime"
```

Reply:
uh ok, thats new to me.

Reply:
Yeah, just an oversight. 😁